.github/workflows/integration-test.yml

```diff
@@ -0,0 +1,70 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Run Integration Tests
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  integration-test:
+
+    runs-on: ubuntu-latest
+
+    services:
+      postgres:
+        image: postgres:9.6
+        env:
+          POSTGRES_PASSWORD: postgres
```

Comment:
Could there a potential concern (now or in the future) having passwords here in plain text?

Reply:
I am inclined to say that in this case it's okay, we've sort of set a standard for doing it with throwaway DBs like this one. The nature of this one isn't really "secret"